Servlets/src/main/java/GetNameServlet.java

```diff
@@ -0,0 +1,21 @@
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.io.PrintWriter;
+
+public class GetNameServlet extends HttpServlet {
+    protected void doGet(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
+
+        PrintWriter writer = response.getWriter();
+        // получаем параметр id
+        String name = request.getParameter("name");
+
+        try {
+            writer.println("Name is " + name);
```

Comment:
If you want to implement  the complete auto-closeabl pattern, use try-with-resources and rethrow an exception caught (or just log it).
Otherwise, this "try without catch" doesn't actually have sense